.eslintrc.js

```diff
@@ -1,28 +1,28 @@
 module.exports = {
   env: {
     browser: true,
-    commonjs: true,
-    es2021: true
+    es2021: true,
   },
-  extends: ['plugin:react/recommended', 'airbnb', 'prettier', 'plugin:jsdoc/recommended'],
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
+  ],
+  overrides: [
+  ],
   parserOptions: {
-    ecmaFeatures: {
-      jsx: true
-    },
-    ecmaVersion: 14
+    ecmaVersion: 'latest',
+    sourceType: 'module',
   },
-  plugins: ['prettier', 'jsdoc'],
+  plugins: [
+    'react',
+  ],
   rules: {
```

Comment:
Just to clarify here: everything under the rules section are turned off. They are rules included in the airbnb distribution, and this code disables them.